common/openapi.yml

```diff
@@ -751,6 +751,10 @@ components:
           items:
             type: string
             format: uuid
+        errors:
+          type: array
+          items:
+            type: string
```

Comment:
It seems in other places in this file `errors` is just a string instead of Array of string. I think we should make it same as others. What do you think?